Defra.Cdp.Backend.Api/Program.cs

```diff
@@ -53,8 +56,45 @@
 
 // Add health checks and http client
 builder.Services.AddHealthChecks();
-builder.Services.AddHttpClient();
-builder.Services.AddHttpProxyClient(logger);
+
+builder.Services.AddHttpClient("DefaultClient", client =>
+    {
+        client.DefaultRequestHeaders.Accept.Clear();
+        client.DefaultRequestHeaders.Add(
+            "Accept", "application/json");
+        client.DefaultRequestHeaders.Add("User-Agent",
+            "cdp-portal-backend");
+    })
+    .AddHeaderPropagation();
+
+builder.Services.AddHttpClient("GitHubClient", client =>
+{
+    client.DefaultRequestHeaders.Accept.Clear();
+    client.DefaultRequestHeaders.Add(
+        "Accept", "application/vnd.github+json");
+    client.DefaultRequestHeaders.Add("X-GitHub-Api-Version", "2022-11-28");
+    client.DefaultRequestHeaders.Add("User-Agent",
+        "cdp-portal-backend"); // required by GitHub API or else you get 403
+}).ConfigurePrimaryHttpMessageHandler(() => Proxy.ProxyHttpMessageHandler(logger)).AddHeaderPropagation();
```

Comment:
We can remove the need to pass the logger instance here:
```
.ConfigurePrimaryHttpMessageHandler(provider =>
            {
                var loggerFactory = provider.GetService<ILoggerFactory>();
                return Proxy.ProxyHttpMessageHandler(loggerFactory.CreateLogger("GithubClient"));
            })
            ```
Would need to swap code in proxy to use normal logger, not serilog but that's easily done,